egs/wsj/s5/local/nnet3/run_lstm.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# this is a basic lstm script
+# LSTM script runs for more epochs than the TDNN script
+# and each epoch takes twice the time
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call lstm/train.sh with --gpu false
+
+stage=0
+train_stage=-10
+affix=
+common_egs_dir=
+
+# LSTM options
+splice_indexes="-2,-1,0,1,2 0 0"
+lstm_delay=" -1 -2 -3 "
+label_delay=5
+num_lstm_layers=3
+cell_dim=1024
+hidden_dim=1024
+recurrent_projection_dim=256
+non_recurrent_projection_dim=256
+chunk_width=20
+chunk_left_context=40
+chunk_right_context=0
+
+
+# training options
+num_epochs=10
+initial_effective_lrate=0.0006
```

Comment:
I think we use smaller learning rates in AMI and SWBD scripts. Is there a reason for choosing these learning rates?